.changeset/biome-deps-update.md

```diff
@@ -0,0 +1,5 @@
+---
+"@technance/code-style": patch
+---
+
+Add Biome as dev dependency and update ESLint-related dependencies to latest versions
```

Comment:
Let's do a minor version bump.

As per [semver](https://semver.org/) standard:

> MINOR version when you add functionality in a backward compatible manner


Steps:

1. Delete this changeset
2. Recreate a new changeset via `pnpm changeset`
3. Choose minor

Also for the message of changelog use:

"Update all dependencies to latest version"

Since this is something for the end users. [Changelog is for the end users of the software.](https://keepachangelog.com/)